tests/various/struct_access.ys

```diff
@@ -0,0 +1,4 @@
+read_verilog -sv struct_access.sv
+hierarchy; proc; opt
```

Comment:
Can we split these passes onto separate lines for better visibility?

Reply:
Done.